src/main/java/com/google/sps/data/ChromeOSDevice.java

```diff
@@ -104,4 +104,7 @@ public String toString() {
     return "[Device with ID: " + deviceId + "]";
   }
 
+  public String getCSVLine() {
+    return deviceId + "," + serialNumber + "," + annotatedAssetId + ","  + annotatedLocation + ","  + annotatedUser + "\n";  
```

Comment:
I think we should escape comma and quote for the user editable fields, otherwise the csv will not parse correctly if there's comma in those values.

Reply:
+1. You may use some Java library to construct the CSV, e.g. https://commons.apache.org/proper/commons-csv/